internal/sync/sync.go

```diff
@@ -81,21 +83,11 @@ func syncRemoteRepo(repo *repo) {
 	}
 }
 
-func repoSynced(clonedDir, localDir string, repo *repo) bool {
-	glog.V(100).Infof("Verifying destination directory %s exists", localDir)
-
-	if _, err := os.Stat(localDir); os.IsNotExist(err) {
-		glog.V(100).Infof("Destination api directory %s doesn't exist creating directory", localDir)
-
-		if os.MkdirAll(localDir, 0777) != nil {
-			glog.V(100).Infof("Failed to create api directory. Exit with error code 1")
-			os.Exit(1)
-		}
-	}
+func excludeAndRefactor(clonedDir, localDir string, repo *repo) {
```

Comment:
Please either add comment to this function:
// Remove excludes files and refactor code in cloned directory. 

or re-name it to excludeFilesAndRefactorCodeInCloneDirectory